core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -236,7 +236,7 @@ describe('Learner dashboard functionality', function() {
       TOPIC_NAME, TOPIC_URL_FRAGMENT_NAME, TOPIC_DESCRIPTION, false);
     await topicEditorPage.expectNumberOfStoriesToBe(0);
     await topicEditorPage.createStory(
-      'Story Title', 'story-one', 'Story description',
+      'Story Title', 'storyone', 'Story description',
```

Comment:
Why are you changing `story-one` to `storyone`? Ditto elsewhere.

Reply:
Hi @SAEb-ai, actually these changes were made because the schema for StoryUrlFragmentHandler was causing the e2e tests to fail. Due to the fact that the e2e tests simulate slow typing, an error was shown at each hyphen (if full url fragment is `story-one` then at `story-` error was shown). @vojtechjelinek had suggested that the e2e tests be changed in this manner so that the url fragment would not end in a hyphen at any point.

Reply:
Oh I see, yes I will make the changes as soon as possible.

Reply:
@SAEb-ai, actually I tried to convert the url fragments to camelCase, but it is also causing the e2e tests to fail since the regex in the schema does not allow the url fragments to end in capital letter.

Reply:
Oh I see. Thanks for testing.